buildenv/jenkins/variables/defaults.yml

```diff
@@ -681,6 +681,33 @@ x86-64_mac_uma:
 x86-64_mac_xl_uma:
   extends: ['x86-64_mac_xl', 'uma']
 #========================================#
+# Mac M1 Aarch64
+#========================================#
+aarch64_mac:
+  extends: ['boot_jdk_default']
+  boot_jdk:
+    arch: 'aarch64'
+    os: 'mac'
+    dir_strip:
+      all: '3'
+  release:
+    all: 'macosx-aarch64-server-release'
+    8: 'macosx-x86_64-normal-server-release'
+    11: 'macosx-x86_64-normal-server-release'
+  extra_configure_options:
+    8: '--with-toolchain-type=clang --with-freetype=/usr/local/freetype-2.9.1'
+    17: '--disable-warnings-as-errors --with-noncompressedrefs --disable-ddr'
+  openjdk_reference_repo: '/Users/jenkins/openjdk_cache'
+  node_labels:
+    build: 'ci.role.build && hw.arch.aarch64 && sw.os.mac'
+#  build_env:
```

Comment:
Might as well remove these 2 commented out lines.  Unless there is a specific need to have it that is.